ektelo/wrapper.py

```diff
@@ -56,8 +56,8 @@ def laplace_scale_factor(M, eps):
     return cmeasurement.laplace_scale_factor(M, eps)
 
 
-def non_negative_least_squares(Ms, ys, scale_factors=None):
-    return inference.NonNegativeLeastSquares().infer(Ms, ys, scale_factors)
+def non_negative_least_squares(Ms, ys, l1_reg = 0, l2_reg =0, scale_factors=None):
```

Comment:
Back in standalone.py, l1_reg and l2_reg were set to 1e-6. What's the reasoning behind the choices? Should this be documented?

Reply:
The default regs for the inference operator are still 0. In standalone, MWEM variants are using non-zero regs for nnls. It's plan-level parameter choice to avoid insanely long converging time.